docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contribution Guidelines
+
+As all source{d} projects, this project follows the
+[source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md).
+
+
+## Additional Contribution Guidelines
+
+In addition to the [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md),
+this project follows the guidelines described below.
+
+
+# Architecture
+
+The application is a go binnary that serves the statics and the API used for the UI.
+
+The statics (`html`, `js` and `css`) are generated by `yarn`, embedded in the go binary using bindata, and served from the binary itself.
+
+There are two main different ways to run the app:
+
+- using released artifacts, as [listed in the README.md#](../README.md#run-the-playground)
+- serving the app for development purposes, [building and running it from sources](#run-from-sources)
+
+
+# Configuration
```

Comment:
Maybe this section should be moved to the quickstart guide? These env vars are also relevant to people interested in running the released binaries or docker.